assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,45 @@
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, covers_view, row, loadMoreCovers }) => {
+  const rowAmount = 3;
+  const showLoadMore = (covers.length > rowAmount && covers_view === '1') ||
+    (covers.length > (rowAmount * 2) && covers_view === '2');
+
+  return (
+    <div className='container'>
+      <div className='thumbnail-largeview-container'>
+        {covers.map((cover, index) => {
+          const { href, image, alt_text, name } = cover;
+          const hideCover = covers_view !== '3' && index >= row * rowAmount;
+          return (
+            <div key={name} className={`campaign-card-column ${hideCover ? 'hidden' : ''}`}>
+              <a
+                href={href}
+                data-ga-category='Campaign Covers'
+                data-ga-action='Image'
+                data-ga-label='n/a'
+                aria-label={__('Check our campaign about ' + name, 'planet4-blocks')}
+              >
+                <div className='thumbnail-large'>
+                  {image && image[0] &&
+                    <img loading='lazy' src={image[0]} alt={alt_text} />
```

Comment:
Images always need to have an `srcset`

Reply:
After you added an `srcset` you can get the `sizes` attribute with this bookmarklet https://ausi.github.io/respimagelint/ . You can right click and "copy link adress" on the big button, and add it as a bookmark in your browser. Then you can open a page with a covers block and click the bookmarklet. It will in the background resize the current page to all possible screen sizes and send you to a report page where it suggests the `sizes` attribute. Probably won't work in a private tab (it doesn't for me), but regular Chrome it should.

Reply:
Added! 🙂